dev-envs/linux/Dockerfile

```diff
@@ -0,0 +1,76 @@
+ARG ARCH=arm64
+ARG TAG=latest
+FROM datadog/agent-buildimages-deb_${ARCH}:${TAG}
+
+# Set up scripts
+COPY scripts /root/.scripts
+COPY scripts.sh /setup/scripts.sh
+RUN /setup/scripts.sh
+
+# We set up shells first so that config can be modified as part of subsequent steps.
+# Shells require scripts to set environment variables and add to PATH. These same
+# scripts require the shells to be set up, leaving us with a cyclical dependency.
+# To break this cycle, each shell's step adds required setup logic to a file that
+# is sourced before globally making modifications to PATH for shells.
+COPY env.sh /setup/env.sh
+COPY <<-"EOF" /setup/shellrc.sh
+set -euxo pipefail
+
+export PATH="${HOME}/.scripts:${PATH}"
+EOF
+
+# Set up Zsh
+COPY zsh.sh /setup/zsh.sh
+RUN /setup/zsh.sh
```

Comment:
Even if it's much more readable/"debuggable" like that, should we have grouped `COPY` or `RUN` commands to have a lighter image?